cmd/root.go

```diff
@@ -97,18 +100,30 @@ func startGohaqd(cmd *cobra.Command, args []string) {
 		QueueUrl:        q.QueueUrl,
 		WaitTimeSeconds: aws.Int64(20),
 	}
+	sem = make(chan *sqs.Message)
```

Comment:
@ameykpatil I had taken that approach first, but changed it since it doesn't really help. Consumers are almost always slower than producers here.

And it increases the chances of same messages being consumed buy other instances as well. If messages stay in buffer for longer than the SQS visibility timeout, they'd be picked by other gohaqd instances as well.

Does this make sense?